### PR TITLE
SEO-171754-Windowsforms controls

### PR DESCRIPTION
**Pull request by Edithabich**
Hi @Aishwarya-Ganesan ,

Have made the changes on redirect links. Please review.

Thanks.

Regards,
Edith.

### CODE REVIEW

File: WindowsForms/Grid-Control/MS-Excel-Like-Features.md [hunk 1]
```diff
@@ -463,7 +463,7 @@ N> Any range of cells cannot be merged like the covered ranges because merging h
...
+Floating cells  are those cells whose content floats over empty, adjacent cells. The floating cells can be enabled by setting [GridControl.FloatCellsMode](https://help.syncfusion.com/cr/windowsforms/Syncfusion.Windows.Forms.Grid.GridControl.html#Syncfusion_Windows_Forms_Grid_GridControl_FloatCellsMode) property. Setting this property to[GridFloatCellsMode.BeforeDisplayCalculation](https://help.syncfusion.com/cr/windowsforms/Syncfusion.Windows.Forms.Grid.GridFloatCellsMode.ht) will force the floating cells to always be calculated just prior to being displayed. Setting the property to [GridFloatCellsMode.OnDemandCalculation](https://help.syncfusion.com/cr/windowsforms/Syncfusion.Windows.Forms.Grid.GridFloatCellsMode.html) will calculate the floating cells only if the cell contents or size changes.
```
Comment (Yvone-Atieno) [inline WindowsForms/Grid-Control/MS-Excel-Like-Features.md:466]: Recheck the below link,
image

File: WindowsForms/Grid-Control/TouchSupport.md [hunk 2]
```diff
@@ -28,7 +28,7 @@ N> To have the proper horizontal and vertical pixel touch scrolling, the [HScrol
 
 ## Touch Zooming
 GridControl allows to perform the touch zoom in or out to the entire grid by initializing [ZoomGrid](https://help.syncfusion.com/cr/windowsforms/Syncfusion.GridHelperClasses.ZoomGrid.html#) class where the constructor takes a parameter as a `GridControl` object.
-To have this touch zooming support in grid, the [GridHelperClasses.Windows](http://help.syncfusion.com/cr/windowsforms/Syncfusion.GridHelperClasses.html#) assembly needs to be added into the assembly reference. 
+To have this touch zooming support in grid, the [GridHelperClasses.Windows](https://help.syncfusion.com/cr/windowsforms/Syncfusion.GridHelperClassesWindowsAssembly.html) assembly needs to be added into the assembly reference.
```
Comment (Yvone-Atieno) [inline WindowsForms/Grid-Control/TouchSupport.md:31]: Recheck this link,
https://help.syncfusion.com/cr/windowsforms/Syncfusion.GridHelperClassesWindowsAssembly.html

File: WindowsForms/Grid-Control/MS-Excel-Like-Features.md (hunk 1 above)
Comment (Edithabich) [reply WindowsForms/Grid-Control/MS-Excel-Like-Features.md]: I have checked and updated proper redirect link.

File: WindowsForms/Grid-Control/TouchSupport.md (hunk 2 above)
Comment (Edithabich) [reply WindowsForms/Grid-Control/TouchSupport.md]: I have rechecked and updated proper link

**Review by gowthamanthavasiyappan — APPROVED** [review]
Approved

**Review by Girija-Rajendran — APPROVED** [review]

**Pull request merged**